topics/statistics/tutorials/classification_machinelearning/tutorial.md

```diff
@@ -501,7 +502,6 @@ After extracting the parameter names from the **Pipeline builder** file, we will
 >        - {% icon param-files %} *"Choose the dataset containing pipeline/estimator object"*: `h5mlm` file (output of **Pipeline builder** {% icon tool %})
 >        - {% icon param-files %} *"Is the estimator a deep learning model?"*: `NO` {% icon tool %})
 >        - In *"Search parameters Builder"*:
->             - {% icon param-files %} *"Choose the dataset containing parameter names"*: `tabular` file (output of **Estimator attributes** {% icon tool %})
```

Comment:
Can they just not set this file? Is there no need to specify that this data will now be used from the `h5mlm` file?

Reply:
Ahh, so there is also within the tool itself no need to set the location/file any more where to find the hyperparameters. Thanks for clarifying.